server/commands/battle.py

```diff
@@ -0,0 +1,597 @@
+                                        args[i].lower()
+                                    )
+                            with open(
+                                "storage/battlesystem/battlesystem.yaml",
+                                "w",
+                                encoding="utf-8",
+                            ) as c_save:
+                                yaml.dump(char, c_save)
+                                client.battle = ClientManager.Battle_char(
+                                    client, fighter
+                                )
+                                client.send_ooc(f"{args[0]} has been added!")
+                        else:
+                            client.send_ooc("This move has already been created.")
+                else:
+                    client.send_ooc("Move's Type should be atk or spa!")
+            else:
+                client.send_ooc("Accuracy should be a integer between 1 and 100")
+        else:
+            client.send_ooc("Power argument is not an integer.")
+    else:
+        client.send_ooc(
+            "You have to choose a figher to create a move.\n /choose_fighter FighterName"
+        )
+
+
+@mod_only(hub_owners=True)
+def ooc_cmd_modify_stat(client, arg):
+    """
+    Allow you to modify fighter's stats.
+    Usage: /modify_stat FighterName Stat Value
+    """
+    args = arg.split(" ")
+    with open(
+        "storage/battlesystem/battlesystem.yaml", "r", encoding="utf-8"
+    ) as c_load:
+        char = yaml.safe_load(c_load)
+        if args[0].lower() in char:
+            if args[1].lower() in ["hp", "atk", "def", "spa", "spd", "spe"]:
+                if args[2].isdigit and float(args[2]) > 0:
+                    char[args[0].lower()][args[1].upper()] = float(args[2])
+                    with open(
+                        "storage/battlesystem/battlesystem.yaml", "w", encoding="utf-8"
+                    ) as c_save:
+                        yaml.dump(char, c_save)
+                        client.send_ooc(
+                            f"{args[0]}'s {args[1]} has been modified. To check the changes choose again this fighter"
+                        )
+                else:
+                    client.send_ooc("The value have to be an integer greater than zero")
+            else:
+                client.send_ooc(
+                    "You could just modify this stats:\nhp, atk, def, spa, spd, spe"
+                )
+        else:
+            client.send_ooc("No fighter has this name!")
+
+
+@mod_only(hub_owners=True)
+def ooc_cmd_delete_fighter(client, arg):
+    """
+    Allow you to delete a fighter.
+    Usage: /delete_move FighterName
+    """
+    with open(
+        "storage/battlesystem/battlesystem.yaml", "r", encoding="utf-8"
+    ) as c_load:
+        char = yaml.safe_load(c_load)
+        if arg.lower() in char:
+            char.pop(arg.lower())
+            with open(
+                "storage/battlesystem/battlesystem.yaml", "w", encoding="utf-8"
```

Comment:
wait why are fighters saved into battlesystem.yaml and not their own yaml files named after the character?

Reply:
Done!